CMakeLists.txt

```diff
@@ -336,14 +336,12 @@ Enable this if the module uses typeid or dynamic_cast. You will probably need to
     add_subdirectory(sdk/include/reactos/mc)
     add_subdirectory(sdk/include/asm)
 
-    if(NO_ROSSYM)
+    if(ARCH MATCHES "64$")
```

Comment:
Would this affect in the future in AArch64? Or if we have anymore 64 bits arch?

Reply:
Everything that ends with '64' in the arch name. "amd64", "arm64"; "mips64", "riscv64", "loongarch64", "elbrus64", you name it.